views/setup-wizard/shop-name.php

```diff
@@ -5,11 +5,64 @@
 </div>
 <div class="wpo-setup-input">
 	<?php
+	// Set default values for current setting to be used in case the user has not set them yet.
 	$current_settings = wp_parse_args( get_option( 'wpo_wcpdf_settings_general', array() ), array(
-		'shop_name'    => array( 'default' => get_bloginfo( 'name' ) ),
-		'shop_address' => array( 'default' => '' ),
+		'shop_name'             => array( 'default' => get_bloginfo( 'name' ) ),
+		'shop_address_line_1'   => array( 'default' => get_option( 'woocommerce_store_address' ) ),
+		'shop_address_line_2'   => array( 'default' => get_option( 'woocommerce_store_address_2' ) ),
+		'shop_address_country'  => array( 'default' => get_option( 'woocommerce_store_country' ) ?? '' ),
```

Comment:
You could use the second parameter of the `get_option()` to set the default value, no need to use `??`.